src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1622,6 +1662,44 @@ private void RemoveOrganelleAt(Hex location)
         EnqueueAction(action);
     }
 
+    [DeserializedCallbackAllowed]
+    private void DoOrganelleMoveAction(MicrobeEditorAction action)
+    {
+        var data = (MoveActionData)action.Data;
+        editedMicrobeOrganelles.Remove(data.Organelle);
+        data.Organelle.Position = data.NewLocation;
+        editedMicrobeOrganelles.Add(data.Organelle);
+        data.Organelle.NumberOfTimesMoved++;
+    }
+
+    [DeserializedCallbackAllowed]
+    private void UndoOrganelleMoveAction(MicrobeEditorAction action)
+    {
+        var data = (MoveActionData)action.Data;
+        editedMicrobeOrganelles.Remove(data.Organelle);
+        data.Organelle.Position = data.OldLocation;
+        editedMicrobeOrganelles.Add(data.Organelle);
+        data.Organelle.NumberOfTimesMoved--;
+    }
+
+    private void MoveOrganelle(Hex oldLocation, Hex newLocation)
```

Comment:
I think passing in here the organelle to be moved would be nice. Also shouldn't this maybe take in the new rotation otherwise you couldn't rotate organelles? Rotating organelles in-place (so only rotation changes but not position) should probably be free.